src/main/java/com/laserfiche/repository/api/clients/params/ParametersForCancelOperation.java

```diff
@@ -12,6 +15,12 @@ public class ParametersForCancelOperation {
      */
     private String operationToken;
 
+    /**
+     * Builder for setting the repoId parameter.
```

Comment:
I am not sure if it is good to mention Builder here, as we are not actually following builder pattern. I mean, when using Builder pattern, something like this is expected 
new ParametersForCancelOperationBuilder().setter1().setter2().setter3()....build();
but we provide no build method, and the client needs to explicitly call the constructor, like
new ParametersForCancelOperation().setter1().setter2()...

Reply:
Update the description to use the following format
`Sets the value of the __ parameter and returns the current object, to enable chaining further setters.`